### ISSUE TITLE
Remove thumbnail saving 

### ISSUE THREAD

**Issue by DominicOram** [DiamondLightSource/dodal#633]
Once https://jira.diamond.ac.uk/browse/LIMS-1170 is done we will no longer need to save thumbnails in oav/webcam images. We should remove this and the associated code (e.g. everything in https://github.com/DiamondLightSource/dodal/pull/632

## Acceptance Criteria
* `save_thumbnail` no longer exists

**Comment by DominicOram** [comment]
Duplicate of https://github.com/DiamondLightSource/dodal/issues/655
